cetmix_tower_server/tests/test_command.py

```
@@ -0,0 +1,1911 @@
+                "code": """
+server_name = {{ tower.server.name }}
+if server_name and #!cxtower.secret.FOLDER!# == "secretFolder":
+    # We don't actually create a new command because it will raise
+    # access error if user doesn't have access to 'create' operation.
+    # Instead we just return a dummy command result.
+    command = "new command"
+    result = {"exit_code": 0, "message": "New command was created"}
+else:
+    result = {"exit_code": %s, "message": "error"}
+    """
+                % GENERAL_ERROR,
```

Ruff (0.14.5)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)